pkg/graveler/graveler.go

```diff
@@ -1176,7 +1414,36 @@ func (g *KVGraveler) LoadCommits(ctx context.Context, repositoryID RepositoryID,
 }
 
 func (g *KVGraveler) LoadBranches(ctx context.Context, repositoryID RepositoryID, metaRangeID MetaRangeID) error {
-	return g.db.LoadBranches(ctx, repositoryID, metaRangeID)
+	repo, err := g.GetRepository(ctx, repositoryID)
+	if err != nil {
+		return err
+	}
+	// TODO (niro): Replace by KV committedManager
```

Comment:
🤔 
Why does committed manager differ between KV and DB?

Reply:
It's just a TODO for me to validate the current flow as all parts come togehter